### DISCUSSION TITLE
如何进行本地调试开发？

### DISCUSSION THREAD

**Discussion by Lind-pro** [ant-design/pro-components#3172] [Q&A] [answered]
想修改源码发现每次修改之后demo不生效，只能重新build才生效。但是build的时间也太长了吧。

**Comment by xiefengnian** [comment, marked as answer]
`$ yarn build -w`

**Reply by Lind-pro** [reply]
每次修改了代码都重新build吗？
build时间太长了。。而且是所有包全部build。怎么可以只build执行包呢

**Comment by chenshuai2144** [comment]
你是没有使用yarn 吗，还是大小写有问题。我这里正常热更新还是没啥问题的
